docs/DataWorkflows/FetchingYourData.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
 title: Getting Static Data from PMACS
```

Comment:
I would rename the whole file to reflect that this is specific to PMACS.

Reply:
done